astropy/utils/masked/tests/test_masked.py

```diff
@@ -882,6 +891,15 @@ def test_sum_where(self, axis):
         assert_array_equal(ma_sum.unmasked, expected_data)
         assert_array_equal(ma_sum.mask, expected_mask)
 
+    def test_sum_hash(self):
+        # Only works if no element is masked.
```

Comment:
I suggest moving this comment to the `pytest.raises` block

Reply:
OK, adjusted.